docs/concepts/pools/deploy-a-pool.md

```diff
@@ -0,0 +1,39 @@
+# Pool Creation
+
+## Overview
+
+**Note: Before creating a new pool, make sure there isn't already a similar pool. There is no advantage to fragmenting liquidity.**
+
+Anyone can create a pool on Balancer. WeightedPools can be deployed using the Pool creator Interface that can be accessed on [Mainnet](https://app.balancer.fi/#/ethereum), [Polygon](https://app.balancer.fi/#/polygon), [Arbitrum](https://app.balancer.fi/#/arbitrum), and [Gnosis Chain](https://app.balancer.fi/#/gnosis-chain). It's highly recommended that you deploy a pool on a testnet before deploying on a production network.
```

Comment:
I think the pool creation interface is down right now, has that changed?  It would also be good to link directly to the pool creation interface and/or to provide a screenshot to show where to find the link if you are talking about how to do this through the UI.